postgres/init.sql

```diff
@@ -103,16 +103,22 @@ CREATE TABLE ColorVotes (
 CREATE INDEX colorVotes_user_address_index ON ColorVotes (user_address);
 CREATE INDEX colorVotes_color_key_index ON ColorVotes (color_key);
 
+CREATE TABLE TemplateData (
+  key int PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
+  hash text NOT NULL,
+  data bytea NOT NULL
+);
+
 -- TODO: key -> template_id?
 CREATE TABLE Templates (
-  key integer NOT NULL PRIMARY KEY,
+  key int PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
```

Comment:
I'd like to keep this table the same as it was for now. So these changes can be removed.

Reply:
Contents of `Templates` restored.